services/hanke-service/src/test/kotlin/fi/hel/haitaton/hanke/factory/HankeKayttajaFactory.kt

```diff
@@ -204,10 +237,17 @@ class HankeKayttajaFactory(
                 nimi = "test$i name$i",
                 puhelinnumero = "040555$i$i$i$i",
                 kayttooikeustaso = KATSELUOIKEUS,
+                roolit = roolit,
                 tunnistautunut = tunnistautunut
             )
 
-        fun generateHankeKayttajat(amount: Int = 3): List<HankeKayttajaDto> =
-            (1..amount).map { createDto(it, it % 2 == 0) }
+        fun generateHankeKayttaja(i: Int = 1, vararg roolit: ContactType): HankeKayttajaDto =
```

Comment:
So far, we have used create and save as verbs in factories. Save when the result is saved to DB and create when it's not. These could continue that pattern.